include/definitions.h

```diff
@@ -47,6 +47,18 @@
 #define SERIAL_TX 15
 #define SERIAL_BAUDRATE 115200
 
+/*
+ * CAN Bus
+ * 
+ *  GPIO25  TX
+ *  GPIO26  RX
+ */
+#define CAN_TX (gpio_num_t)25
```

Comment:
Conflicts with #define GPIO_INTERRUPT_PIN 25, should we resolve later...

Reply:
Ok, let's discuss this on Sunday. I think this already came up once. Didn't you resolve it with @garned?